packages/config/config/devices/0x027a/zen23_3.0.json

```
@@ -153,6 +166,27 @@
 					"value": 1
 				}
 			]
+		},
+		"13": {
+			"label": "Report for Disabled Toggle",
+			"description": "Set reporting behavior for disabled physical control.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Switch reports on/off status even if physical and Z-Wave control is disabled (default)",
+					"value": 0
+				},
+				{
+					"label": "Switch doesn't report on/off status when physical (and Z-Wave) control is disabled",
+					"value": 1
+				}
+			]
```

The labels are a bit verbose, maybe like this...
```suggestion
			"label": "Report status when toggle is disabled",
			"description": "Report on/off status when if physical and Z-Wave control is disabled.",
			"valueSize": 1,
			"minValue": 0,
			"maxValue": 1,
			"defaultValue": 0,
			"readOnly": false,
			"writeOnly": false,
			"allowManualEntry": false,
			"options": [
				{
					"label": "Yes",
					"value": 0
				},
				{
					"label": "No",
					"value": 1
				}
			]
```